apps/devhub/tests/test_tasks.py

```diff
@@ -237,6 +237,44 @@ def test_annotate_passed_auto_validation_bogus_result(self, _mock):
                                      "low": 0, "trivial": 0}})
 
 
+class TestTrackValidatorStats(amo.tests.TestCase):
+
+    def setUp(self):
+        super(TestTrackValidatorStats, self).setUp()
+
+        p = mock.patch('validator.validate.validate')
+        self.mock_validate = p.start()
+        self.addCleanup(p.stop)
+
+        p = mock.patch('devhub.tasks.statsd.incr')
+        self.mock_incr = p.start()
+        self.addCleanup(p.stop)
+
+    def test_count_all_successes(self):
+        self.mock_validate.return_value = {"errors": 0}
+        tasks.run_validator('some-file.xpi')
+        eq_(self.mock_incr.mock_calls[0][1][0],
+            'devhub.validator.results.all.success')
```

Comment:
We switched to using [pytest](https://pytest.org), so we now use simple `assert`s:
```assert self.mock_incr.mock_calls[0][1][0] == 'devhub.validator.results.all.success'```